patches/minecraft/net/minecraft/client/renderer/WorldRenderer.java.patch

```diff
@@ -85,7 +85,56 @@
        if (this.field_72777_q.field_71441_e.field_73011_w.func_186058_p() == DimensionType.field_223229_c_) {
           this.func_228444_b_(p_228424_1_);
        } else if (this.field_72777_q.field_71441_e.field_73011_w.func_76569_d()) {
-@@ -2283,8 +2298,8 @@
+@@ -1977,25 +1992,31 @@
+       this.field_175008_n.func_217628_a(p_215319_1_, p_215319_2_, p_215319_3_, p_215319_4_);
+    }
+ 
+-   public void func_184377_a(@Nullable SoundEvent p_184377_1_, BlockPos p_184377_2_) {
+-      ISound isound = this.field_147593_P.get(p_184377_2_);
++   @Deprecated // Forge: use item aware function below
++   public void func_184377_a(@Nullable SoundEvent p_184377_1_, BlockPos p_184377_2_)
++   {
```

Comment:
There is no reason to move the bracket onto a new line. (patches are supposed to be as "simple" as possible)